test/index.test.js

```diff
@@ -87,7 +89,7 @@ describe('eslint-config-uphold', () => {
     const source2 = join(dirname, 'fixtures', 'scripts', 'correct.js');
     const results = await linter.lintFiles([source1, source2]);
 
-    results[0].messages.should.be.empty();
-    results[1].messages.should.be.empty();
+    assert.equal(results[0].messages.length, 0);
+    assert.equal(results[1].messages.length, 0);
```

Comment:
We could do 

```suggestion
    const source1 = join(dirname, 'fixtures', 'bin', 'correct.js');
    const source2 = join(dirname, 'fixtures', 'scripts', 'correct.js');
    const [result1, result2] = await linter.lintFiles([source1, source2]);
    
    assert.equal(result1.messages.length, 0);
    assert.equal(result2.messages.length, 0);
```

Or rename them to reference the fixture files.

WDYT?